js/ui/codingManager.js

```diff
@@ -267,7 +267,23 @@ const CodingManager = new Lang.Class({
             this._watchdogId = Mainloop.timeout_add(WATCHDOG_TIME,
                                                     Lang.bind(this, this._watchdogTimeout));
 
-            Util.trySpawn(['flatpak', 'run', 'org.gnome.Builder', '-s', '-m', constructCommand(appManifest)]);
+            let params = new GLib.Variant('(sava{sv})', ['load-flatpak', [new GLib.Variant('s', constructCommand(appManifest))], {}]);
+            Gio.DBus.session.call('org.gnome.Builder',
+                                  '/org/gnome/Builder',
+                                  'org.gtk.Actions',
+                                  'Activate',
+                                  params,
+                                  null,
+                                  Gio.DBusCallFlags.NONE,
+                                  -1,
```

Comment:
Finding javascript's counterpart of the C `G_MAXINT` macro was a bit challenging…